README.md

```diff
@@ -215,6 +215,39 @@ If your Sinatra app consists of subprojects and you want to capture errors
 separately for each subproject, make sure to [configure it
 accordingly](#configuring-individual-notifier-for-each-subproject).
 
+### Hanami
+
+To use Airbrake with specific [hanami](http://hanamirb.org) app you need
+to create initializer file and `use` our [Rack middleware](#rack) in app
+configuration.
+
+```ruby
+# config/initializers/airbrake.rb
+require 'airbrake'
```

Comment:
Seems like this is not needed. We probably should mention that we need to edit project's Gemfile:

```
gem 'airbrake', '~> 5.6'
```